solara/website/pages/documentation/examples/utilities/todo.py

```diff
@@ -183,14 +136,16 @@ def TodoStatus():
         solara.Success("All done, awesome!", dense=True)
 
 
+state = State(initial_items)
```

Comment:
It would be nice with a comment explaining how the state object can still work for multiple clients (virtual kernels), despite it not being declared a solara.reactive.